pybossa/view/projects.py

```diff
@@ -358,15 +358,17 @@ def project_cat_index(category, page):
 def new():
     ensure_authorized_to('create', Project)
 
-    form = dynamic_project_form(ProjectForm, request.body, data_access_levels,
-                                products=current_app.config.get('PRODUCTS_SUBPRODUCTS', {}))
+    # Sort list of subproducts (value) for each product (key).
+    prodsubprods = {key:sorted(value) for key, value in current_app.config.get('PRODUCTS_SUBPRODUCTS', {}).items()}
```

Comment:
Actually why not sorting this products at dynamic_form.py instead? That way will need to be done in one place only. what do you think ?

Reply:
The intention was since the products list was loaded from `current_app.config.get()` within this section, the replacement (and sorting) would be added to the same location, rather than modifying a different file. Thanks. :)

Reply:
sounds good